yarn.lock

```diff
@@ -11487,6 +11487,10 @@ validate-npm-package-license@^3.0.1:
     spdx-correct "~1.0.0"
     spdx-expression-parse "~1.0.0"
 
+value-matcher@^0.1.0:
+  version "0.1.0"
+  resolved "https://registry.yarnpkg.com/value-matcher/-/value-matcher-0.1.0.tgz#4e42230385ca4fb90fc2c9c7f5e0f8a89ace1048"
+
```

Comment:
Why this change?

Reply:
it's been added when i upgraded the module  `value-matcher`

Reply:
But I thought you needed v0.2.0 for the `__and` feature? Plus, I thought @julien-f said internal dependencies should not appear in yarn.lock. So, in my understanding, you should be good as long as xo-web's package.json says `0.2.0`, which is the case since you're up to date with master and 3e5af9e894dab806a56a2ce46e610cc878b72c1b  added it.
@julien-f can you confirm all of the above?

Reply:
@badrAZ I think if you just update to `master`, it will disappear.

Reply:
If it's not enough, just re-run `yarn`.